<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1551</width>
    <height>867</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="add_tc_button">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="add_result_button">
           <property name="text">
            <string>Add Test Result</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="add_tc_button_2">
           <property name="text">
            <string>Add Test Case</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Tag</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="tag_comboBox"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>board</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="board_comboBox"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>bsp</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="bsp_comboBox"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="search_result_button">
            <property name="text">
             <string>Search Test Result</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="result_tableWidget">
         <property name="columnCount">
          <number>12</number>
         </property>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Category</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Sub Category</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Title</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Status</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Domain</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Pre-condition</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Test Sequence</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Pass Criteria</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Linked Work Item</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Comment</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>result</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
